en/docs/get-started/quick-start-guide.md

```diff
@@ -4,20 +4,17 @@ Let's get started with WSO2 Micro Integrator by running a simple integration use
 
 ## Prerequisites
 
-1. Install Java SE Development Kit (JDK) version 11.
+1. Install Java SE Development Kit (JDK) version 11 or 17.
 2. In the environment variables, set the `JAVA_HOME` environment variable.
 
     !!! Info
         For more information on setting the `JAVA_HOME` environment variable for different operating systems, see the [Install and Setup]({{base_path}}/install-and-setup/install/installing-mi) documentation.
 
 3. Go to the [WSO2 Micro Integrator web page](https://wso2.com/integration/micro-integrator/#), click **Download**, and then click **Zip Archive** under the 'Latest Release' section to download the Micro Integrator distribution as a ZIP file.
-4. In the same page click on **Integration Studio** under the 'Other Components' section to download WSO2 Integration Studio.
-
-    !!! Info
-        For more information, see the [installation instructions]({{base_path}}/install-and-setup/install-and-setup-overview/#installing_1).
-
-5. Download the [sample files]({{base_path}}/assets/attachments/quick-start-guide/mi-qsg-home.zip). From this point onwards, let's refer to this directory as `<MI_QSG_HOME>`.
-6. Download [curl](https://curl.haxx.se/) or a similar tool that can call an HTTP endpoint.
+4. Extract the ZIP file. The extracted folder will be referred as the `<MI_HOME>` folder.
+5. In the Visual Studio Code install the Micro Integrator extension.
```

Comment:
Let's point to the install docs https://mi.docs.wso2.com/en/4.3.0/develop/mi-for-vscode/install-wso2-mi-for-vscode/

Reply:
'Install Micro Integrator for VS Code'